doc/ideas/gameplay.md

```diff
@@ -32,6 +32,54 @@ Environment
 ### Forests
 
  - A single villager can hide (per some tile area) in the forest for sneak attacks. Maybe can only visible to a certain type of unit. But invisible to other villagers (until they pop out and start building stuff)
+ - Gaia flock of birds rises up from a tree when startled, when tree is felled or when many people approach and fly off-map, sound indicator hearable a little bit more far (players know whensomeone goes near a woodline at your starting base -> maybe towering?)
+
+### Gaia Units
+
+ - Bear
+   - could have forage sights, farms, and shore fish as locations that it will try to “eat” from
+
+ - Mercenaries could be hired in the wilderness at a camp to act on behalf of a player
+   - not just buying a unit for gold, resources have to be determined
+   - not really another unit to control for a player
+   - could be kind of an extra scout, if you lose yours
+   - or three units relatively weak but in replacement of the militia
+     - you can send them to raid to your oppponent
+
+  - Wolves
+    - will form a pack and will act together
+    - Wolves could wander the map, making efforts to avoid (circle around) “towns”
+    - they should get hungry and seek food (deer, villagers, wild boar, bear, etc.)
+    - while hungry, they should attack in a pack, kill something, and “eat”.
+    - when not hungry, there should be a chance of attack but it should be more likely that the wolves will avoid contact and
+move on
```

Comment:
accidental new line?

Reply:
I don't know what my editor is doing there all the time. Really WTF!